### ISSUE TITLE
ReferenceError: unknown node of type "Literal" with constructor "Object"

### ISSUE THREAD

**Issue by edmorley** [MatAtBread/fast-async#58]
Hi!

A user of Neutrino reported an issue with their project when they updated their dependencies (mozilla-neutrino/neutrino-dev#984) that upon creating a reduced testcase, appears to be due to `fast-async` / `nodent-compiler` regression.

The exception no longer occurs if [nodent-compiler](https://github.com/MatAtBread/nodent-compiler) is rolled back to `v3.1.8` (eg using yarn's `resolutions` feature). The first version of `nodent-compiler` that triggers the exception is `v3.2.0`.

## STR:

1. Clone https://github.com/edmorley/testcase-fast-async-unknown-node
2. `yarn`
3. `yarn babel testcase.js`

## Expected:

Command completes successfully, or else if the testcase's syntax is invalid, a helpful error message is output.

## Actual:

```
ReferenceError: testcase.js: unknown node of type "Literal" with constructor "Object"
    at Generator.print (...\node_modules\babel-generator\lib\printer.js:279:13)
    at Generator.MemberExpression (...\node_modules\babel-generator\lib\generators\expressions.js:229:10)
    at ...\node_modules\babel-generator\lib\printer.js:298:23
    at Buffer.withSource (...\node_modules\babel-generator\lib\buffer.js:159:28)
    at Generator.withSource (...\node_modules\babel-generator\lib\printer.js:189:15)
    at Generator.print (...\node_modules\babel-generator\lib\printer.js:297:10)
    at Generator.AssignmentExpression (...\node_modules\babel-generator\lib\generators\expressions.js:190:8)
    at ...\node_modules\babel-generator\lib\printer.js:298:23
    at Buffer.withSource (...\node_modules\babel-generator\lib\buffer.js:159:28)
    at Generator.withSource (...\node_modules\babel-generator\lib\printer.js:189:15)
```

**Comment by MatAtBread** [comment]
Thanks for reporting this. It's now fixed in `nodent-compiler@3.2.9`. 

Please re-open if this doesn't resolve your case.

**Comment by edmorley** [comment]
This issue no longer reproduces with the new version. Thank you for the fix :-)
